test/parallel/test-preload.js

```diff
@@ -8,9 +8,9 @@ var nodeBinary = process.argv[0];
 
 var preloadOption = function(preloads) {
   var option = '';
-  preloads.forEach(function(preload, index) {
-    // TODO: randomly pick -r or --require
-    option += '-r ' + preload + ' ';
+  preloads.forEach(function(preload) {
+    var optionString = ['-r', '--require'][Math.floor(Math.random()*2)];
```

Comment:
IMO I don't think we should bulk it up to three tests if we can keep it in one - they don't *really* test very different functionality.

Sorry, what do you mean "edge case of using both"?